main.go

```diff
@@ -398,13 +402,13 @@ var testCases = []testRun{
 		},
 	},
 	{
-		name: "histogram with a repeating boundary",
+		name: "histogram with an overlapping boundary",
```

Comment:
```suggestion
		name: "histogram with non-increasing boundaries",
```
The proto calls for the boundaries to be strictly increasing only. Overlapping is a little confusing terminology